docs/content/vectorising.md

```diff
@@ -139,7 +143,7 @@ Dict(fire:MultiDiscrete([2 2 2]), jump:MultiDiscrete([2 2 2]), acceleration:Box(
 ...     "jump": np.array([0, 1, 0]),
 ...     "acceleration": np.random.uniform(-1., 1., size=(3, 2))
 ... }
->>> observations, rewards, dones, infos = envs.step(actions)
+>>> observations, rewards, terminateds, truncateds, infos = envs.step(actions)
```

Comment:
I think we should change `terminateds` and `truncateds` as this isn't good english to `terminations` and `truncations`. Repeat below

Reply:
This is in line with "dones" which also isn't English. I think its common to have "name" + "s" to refer to array of "name"?

Reply:
I agree that "dones" is not english but it is very close to it while "terminated" feels much more unenglish as it uses two suffixes (+ed and +s)